fmriprep/interfaces/images.py

```diff
@@ -122,6 +122,66 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class ConformSeriesInputSpec(BaseInterfaceInputSpec):
+    t1w_list = InputMultiPath(File(exists=True), mandatory=True,
+                              desc='input T1w images')
```

Comment:
The reasoning for this name is in case we need to conform T2w (or any other) images.